cg/cli/post_process/post_process.py

```diff
@@ -35,4 +35,19 @@ def post_process_sequencing_run(context: CGConfig, run_name: str, dry_run: bool)
     post_processing_service.post_process(run_name=run_name, dry_run=dry_run)
 
 
+@post_process_group.command(name="all")
+@DRY_RUN
+@click.pass_obj
+def post_process_all_sequencing_runs(context: CGConfig, dry_run: bool) -> None:
+    """Post-process all sequencing runs from the sequencing instruments."""
+    services: PostProcessingServices = context.post_processing_services
+    are_all_services_successful: bool = True
+    for service in [services.pacbio]:
+        is_service_successful: bool = service.post_process_all(dry_run=dry_run)
+        are_all_services_successful = are_all_services_successful and is_service_successful
+    if not are_all_services_successful:
+        raise click.Abort
```

Comment:
Catching the error here (see comment below) would prevent the bool juggling in line 47